UCM-Powershell-IIS/worker.ps1

```diff
@@ -0,0 +1,208 @@
+<# This is a worker thread process called by UCM-SQL-Monitor.ps1. Below is a declaration
```

Comment:
update name to Worker.ps1 (all filenames are capital) and update calls accordingly.